clients/python-wrapper/README.md

```diff
@@ -45,6 +45,52 @@ main_branch = repo.create(storage_namespace="<storage_namespace>").branch(branch
 ...
 ```
 
+## Examples
+
+### Print sizes of all objects in lakefs://repo/main~2
+
+```py
+ref = lakefs.Repository("repo").ref("main~2")
+for obj in ref.objects():
+  print(f"{o.path}: {o.size_bytes}")
+```
+
+### Difference between two branches
+
+```py
+for i in lakefs.Repository("repo").ref("main").diff("twig"):
+   print(i)
+```
+
+Naturally you could use ref expressions, for instance `.diff("main~2")` also
```

Comment:
If you mean the "main\~2" argument, this should probably say "revisions" instead of "ref expressions" - at least that's what git calls them (i.e. "main", "master" etc. are references, while "main\~2", "main^2" etc. are revisions)

As seen in https://git-scm.com/docs/git-rev-parse#_specifying_revisions.

Reply:
I agree about using Git terminology when possible. My only concern would be breaking existing APIs by renaming fields, etc.

If this causes no existing code (that was guaranteed not to break in 1.0) to break, might as well change it sooner rather than later

Reply:
Unfortunately this is Python. Even argument names cannot change. I don't know how to change "ref".